CONTRIBUTING.md

```diff
@@ -64,46 +78,18 @@ First, make sure you have the prerequisites installed and available on your `$PA
   - Rush will automatically manage the specific version needed by this repo as long as you have any v5 version installed.
   - If you're unable to install a global tool, you can instead call the wrapper script `node <repo root>/common/scripts/install-run-rush.js` any time the guide instructs you to run `rush`. The wrapper script will install a managed copy of Rush in a temporary directory for you.
 
-Next, get the code:
+### Building our repository
 
 1. Fork this repo
 2. Clone your fork locally (`git clone https://github.com/<youruser>/azure-sdk-for-js.git`)
 3. Open a terminal and move into your local copy (`cd azure-sdk-for-js`)
 4. Install and link all dependencies (`rush update`)
+5. Build the code base (`rush rebuild`)
```

Comment:
We should fork the instructions at this point. Step 4 and 5 assume that the package is managed by rush

Reply:
done.